### PR TITLE
Improve a bit README.md

### PR DESCRIPTION
**Pull request by AshtakaOOf**
Add ~~table of contents~~, add Open Collective, and fix some stuff.

### CODE REVIEW

**Comment by AshtakaOOf** [comment]
To see what it looks like https://github.com/AshtakaOOf/PlaceholderMC

**Comment by Scrumplex** [comment]
I don't particularly like table of contents, as GitHub already has one: GitHub's ToC

**Comment by AshtakaOOf** [comment]
You made me learn something new

**Review by Scrumplex — APPROVED** [review]
Thanks!

File: README.md [hunk 1]
```diff
@@ -16,25 +16,24 @@ This is a **fork** of the MultiMC Launcher and not endorsed by MultiMC.
...
+Portable builds are provided for Linux, Windows, and macOS.
 
-For Debian and Arch, you can use these packages for the latest development versions:  
-[![prismlauncher-git](https://img.shields.io/badge/aur-prismlauncher--git-blue)](https://aur.archlinux.org/packages/prismlauncher-git/)
-[![prismlauncher-git](https://img.shields.io/badge/mpr-prismlauncher--git-orange)](https://mpr.makedeb.org/packages/prismlauncher-git)  
-## Help & Support
+For Debian and Arch, you can use these packages to get compiled development versions:  
 
-Feel free to create an issue if you need help. However, you might find it easier to ask in the Discord server.
+[![prismlauncher-git](https://img.shields.io/badge/mpr-prismlauncher--git-orange?style=flat-square&logo=appveyor)](https://mpr.makedeb.org/packages/prismlauncher-git)  [![prismlauncher-git](https://img.shields.io/badge/aur-prismlauncher--git-blue?style=flat-square&logo=appveyor)](https://aur.archlinux.org/packages/prismlauncher-git/)
```
Comment (flowln) [inline README.md]: link to https://prismlauncher.org/wiki/development/development-builds/ or add the MPR / Gentoo ones :D

Comment (AshtakaOOf) [reply README.md]: So I add these at the development build thing?

Comment (AshtakaOOf) [reply README.md]: image
Like this?

Comment (flowln) [reply README.md]: :+1:

**Review by flowln — APPROVED** [review]

**Pull request merged**